### PR TITLE
Add page for Apache Spark solution offer

### PR DESCRIPTION
**Pull request by grobbie**
## Done

- driveby: Correction to scope of Apache Kafka offer
- feature: New page for Apache Spark offer
- feature: Contact form for Apache Spark offer
- feature: integration contact form / page / marketo
- note: adjusted copydoc to change one of the images which was using the old logo. Substituted the image in the page and the copydoc. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12552.demos.haus/data/spark 
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

https://warthogs.atlassian.net/browse/DATAPM-713, https://warthogs.atlassian.net/browse/WD-2075, https://warthogs.atlassian.net/browse/WD-2074

## Screenshots

Sorry no screenshot, but you can run `./run` and navigate to `/data/spark` to review

### CODE REVIEW

File: templates/data/spark/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+{% block meta_description %}Accelerate innovation with the full-stack solution for Apache Spark® from Canonical. 24/7 supported Spark – in the data centre and on the cloud.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1sM8pgyM-N8VK8fyHgMsW21PdXUiqYrdRkPkipIljNd0/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Accelerate AI/ML innovation anywhere with Apache Spark®</h1>
```
Comment (mtruj013) [inline templates/data/spark/index.html:14]:
```suggestion
      <h1>Accelerate AI/ML innovation anywhere with Apache Spark&reg;</h1>
```

File: templates/data/spark/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      <h2>Everything you need to run Spark®</h2>
```
Comment (mtruj013) [inline templates/data/spark/index.html:15]:
```suggestion
      <h2>Everything you need to run Spark&reg;</h2>
```

File: templates/data/spark/index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      <p>Get 24/7 cover wherever you run your operations - run behind the firewall in your own data centre, or on the cloud – with a fully integrated, supported solution for Apache Spark® from Canonical.</p>
```
Comment (mtruj013) [inline templates/data/spark/index.html:16]:
```suggestion
      <p>Get 24/7 cover wherever you run your operations &ndash; run behind the firewall in your own data centre, or on the cloud &mdash; with a fully integrated, supported solution for Apache Spark&reg; from Canonical.</p>
```

File: templates/data/spark/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h2 class="u-sv3">
+        Canonical’s complete, integrated and fully supported open source solution for data engineering and machine learning at scale.
```
Comment (mtruj013) [inline templates/data/spark/index.html:39]:
```suggestion
        Canonical's complete, integrated and fully supported open source solution for data engineering and machine learning at scale.
```

File: templates/data/spark/index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+      </h2>
+      <ul>
+        <li>A complete, integrated, full-stack solution for Apache Spark®</li>
```
Comment (mtruj013) [inline templates/data/spark/index.html:42]:
```suggestion
        <li>A complete, integrated, full-stack solution for Apache Spark&reg;</li>
```

File: templates/data/spark/index.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <h3>No friction</h3>
+      <p>Get started fast &mdash; the open source solution is free to use in any environment, in the data centre and in the cloud.</p>
+      <p><a href="https://snapcraft.io/spark-client" class="p-button">Try our Spark solution now</a></p>
+    </div>
+    <div class="col-4">
+      <h3>Expressway operations</h3>
+      <p>Rapidly deploy your Apache Spark® applications with the integrated, full stack solution from Canonical.</p>
```
Comment (mtruj013) [inline templates/data/spark/index.html:64]:
```suggestion
      <p>Rapidly deploy your Apache Spark&reg; applications with the integrated, full stack solution from Canonical.</p>
```

File: templates/data/spark/index.html [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+      <p>24/7 support and professional services are available to ensure your initiatives are always on track.</p>
+      <p><a href="/data/spark#get-in-touch" class="p-button">Contact sales</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>The best choice for Apache Spark®, anywhere</h2>
```
Comment (mtruj013) [inline templates/data/spark/index.html:78]:
```suggestion
      <h2>The best choice for Apache Spark&reg;, anywhere</h2>
```

File: templates/data/spark/index.html [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+      <p>Canonical’s scalable and comprehensive full stack system for Apache Spark covers:</p>
```
Comment (mtruj013) [inline templates/data/spark/index.html:79]:
```suggestion
      <p>Canonical's scalable and comprehensive full stack system for Apache Spark covers:</p>
```

File: templates/data/spark/index.html [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <h2>Expert services</h2>
+      <p>A production-grade, fully supported Spark environment deployed and operational, no fuss.</p>
+      <ul>
+        <li>Integrated solution on Charmed Kubernetes, Azure, and AWS.</li>
```
Comment (mtruj013) [inline templates/data/spark/index.html:107]: Please adjust the indentation here so it aligns with the rest of the list elements

File: templates/data/spark/index.html [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+    </div>
+  </div>
+</section>
+
+{% with first_item="_server_download", second_item="_server_contact_us", third_item="_cloud_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <p>Apache®, Apache Spark, Spark®, and the Spark logo are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.</p>
```
Comment (mtruj013) [inline templates/data/spark/index.html:121]:
```suggestion
      <p>Apache&reg;, Apache Spark, Spark&reg;, and the Spark logo are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.</p>
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @grobbie! Some comments in the code. Could you please also link to the correct copy doc at the top of the kafka template? It still links to the old one (which you can feel free to delete by the way, since we already have an up to date version in the correct location)

**Comment by lyubomir-popov** [comment]
Hi, this page is mostly consistent with other pages on the site. I noticed a few small things:

- [ ] the registered marks need to be superscript, in all cases where they appear

image

Can we please style the following as an h4: (`.p-heading--4`)
- [ ] "Everything you need to run Spark®" 
- [ ] image

**Review by mtruj013 — APPROVED** [review]
Thanks @grobbie! LGTM 👍🏻

**Pull request merged**